BFE_RShiny/oasisui/R/exposurevalidation_map_module.R

```diff
@@ -437,7 +439,21 @@ exposurevalidationmap <- function(input,
   # Drawn circles infos, outputs and radius
   .DrawnCircles <- function(radius, lat_click, long_click, ratio) {
 
-    #get pins coordinates
+    # calculate LocID and TIV for pins inside areas
+    locID_list <- .is_within_bounds(result$uploaded_locs_check_peril$LocNumber, radius, lat_click, long_click, ratio)
```

Comment:
it seems you lost the `format()` here:
`format(result$uploaded_locs_check_peril$LocNumber[x], big.mark = "", scientific = FALSE)`

not sure whether that matters and you should put it back?